common/landed_titles/01_landed_titles.txt

```diff
@@ -10866,7 +10866,7 @@ e_great_barrens = {
 					color2 = { 255 255 255 }
```

Comment:
So I would blur coastline, it's too sharp.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/135163232-2ab18f63-3677-465d-989c-66b2b9ecd0ad.png)

</details>

Reply:
Resolved in e0982e4.